Bottom: 72a87fcf4b153e9e3d3820b7e0825a0f5b148ea5
Top:    f5dc80b55c72a199cab3ec85a3eeff0800c7d994
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-13 21:38:19 +0000

Refresh of more-changes

---

diff --git a/src/polygon.h b/src/polygon.h
index 4f9e9c6..94e1a86 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 20 //40 //8 //40
+#define POLY_CIRC_SEGS 40 //8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
diff --git a/src/polygon1.c b/src/polygon1.c
index 9ed6a20..7561400 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1376,19 +1376,6 @@ intersect (jmp_buf * jb, POLYAREA * b, POLYAREA * a, int add)
   return 0;
 }
 
-#if 0
-static void
-intersect_rounded_pl_m_pl (jump_buf * e, POLYAREA *outer_contour, POLYAREA *m_contour)
-{
-  PLINE *pl;
-
-  for (pl = m_contour; pl != NULL; pl = pl->next)
-    {
-      intersect_rounded_no_trees (e, outer_contour, pl, true);
-    }
-}
-#endif
-
 static void
 M_POLYAREA_intersect (jmp_buf * e, POLYAREA * afst, POLYAREA * bfst, int add, CVCList **list_out)
 {
@@ -3299,7 +3286,7 @@ Touching (POLYAREA * a, POLYAREA * b)
       if (!poly_Valid (b))
 	return -1;
 #endif
-      M_POLYAREA_intersect (&e, a, b, false, NULL);
+//      M_POLYAREA_intersect (&e, a, b, false, NULL);
 
       if (M_POLYAREA_label (a, b, TRUE))
 	return TRUE;
@@ -3356,7 +3343,7 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
 
   *res = NULL;
 
-#if 1
+#if 0
   /* Make copies for tracking polygon parentage (DEBUG) */
   if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
       return err_no_memory;
